src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -114,6 +114,12 @@ public static PackageSpec GetPackageSpec(Stream stream, string name, string pack
             packageSpec.Summary = rawPackageSpec.GetValue<string>("summary");
             packageSpec.ReleaseNotes = rawPackageSpec.GetValue<string>("releaseNotes");
 
+            var buildOptions = rawPackageSpec["buildOptions"] as JObject;
```

Comment:
Are there any other `buildOptions`? If so, it may be worth having a new type called `BuildOptions` so the C# reflects the JSON.